inc/utype.h

```diff
@@ -145,6 +147,35 @@ extern struct arabicforms {
     unsigned int required_lig_with_alef: 1;
 } ArabicForms[256];	/* for chars 0x600-0x6ff, subtract 0x600 to use array */
 
```

Comment:
I'm puzzled how ArabicForms is involved here. Maybe something in the future, but at this point, this PR has no connection.